### PR TITLE
[SRVCOM-3387] Integrations - add mention of supported OSSM version

### PR DESCRIPTION
**Pull request by ochromy**
Version(s):
serverless-docs-1.35+

Issue:
https://issues.redhat.com/browse/SRVCOM-3387

Link to docs preview:
[Integrating Service Mesh with OpenShift Serverless](https://84870--ocpdocs-pr.netlify.app/openshift-serverless/latest/integrations/serverless-ossm-setup.html)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by mgencur** [comment]
LGTM from QE

**Review by dfitzmau — APPROVED** [review]

File: integrations/serverless-ossm-setup.adoc [hunk 1]
```diff
@@ -6,6 +6,13 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+[IMPORTANT]
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply integrations/serverless-ossm-setup.adoc:9]: Ah here is the guidance around this. I knew some guidance existed somewhere: https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc


Screenshot from 2024-11-28 15-41-08

[the remark this answers is not in the document]
Comment (ochromy) [reply integrations/serverless-ossm-setup.adoc:11]: Moved, thank you!

File: integrations/serverless-ossm-setup.adoc [hunk 2]
```diff
@@ -8,6 +8,13 @@ toc::[]
 
 The {ServerlessOperatorName} provides Kourier as the default ingress for Knative. However, you can use {SMProductShortName} with {ServerlessProductName} whether Kourier is enabled or not. Integrating with Kourier disabled allows you to configure additional networking and routing options that the Kourier ingress does not support, such as mTLS functionality.
 
+[IMPORTANT]
+====
+Integration of {SMProductName} 3 with {ServerlessProductName} is not supported.
```
Comment (xenolinux) [inline integrations/serverless-ossm-setup.adoc:13]:
```suggestion
Integration of {SMProductName} 3.0 with {ServerlessProductName} is not supported.
```
when talking about the specific release, it's best to use version.X.

**Review by xenolinux — COMMENTED** [review]
I have one suggestion but I will go ahead and merge the PR.

**Pull request merged**